pytest_tests/tests/services/rest_gate/test_rest_gate.py

```diff
@@ -335,11 +335,10 @@ def test_put_http_get_http(self, complex_object_size, simple_object_size, gw_par
     @pytest.mark.parametrize(
         "attributes",
         [
-            {"fileName": "simple_obj_filename"},
-            {"file-Name": "simple obj filename"},
-            {"cat%jpeg": "cat%jpeg"},
```

Comment:
You're removing it without a notice, it should be a separate commit. https://github.com/nspcc-dev/neofs-rest-gw/issues/195 is the real reason (that we can't fix now).

Reply:
I thought it was an officially  unsupported symbol in current rest gw, returned it back and skipped properly.